frontend/src/PatientApp.tsx

```diff
@@ -29,6 +27,20 @@ const styles = (theme: Theme) =>
     },
     title: {
       flexGrow: 1,
+      fontFmily: "Roboto",
+      fontStyle: "normal",
+      fontWeight: "bold",
+      fontSize: "18px",
+      letterSpacing: "0.03em",
+      fontVariant: "small-caps",
+    },
+    actionButton: {
+      backgroundColor: "#FCD446",
```

Comment:
is there a standard material color we can use?

Reply:
Dunno, but Susanne is looking into MUI.